transports/quartz/src/main/java/org/mule/transport/quartz/QuartzMessageReceiver.java

```diff
@@ -83,8 +83,15 @@ protected void doStart() throws MuleException
     {
         try
         {
+
             Scheduler scheduler = connector.getQuartzScheduler();
 
+            if (scheduler.isInStandbyMode())
```

Comment:
Previously the doStop method from QuartzConnector set the scheduler in Standby Mode. I've changed it to shutdown according to your request. The difference is that in shutdown all resources are freed and it cannot be restarted without recreating the scheduler (now if you send a stop request from mmc and then you send a start request, the scheduler is recreated and not restarted because it was formerly shutdown).